test/functional/rpc_verifychainlock.py

```diff
@@ -65,10 +65,15 @@ def run_test(self):
         tx1 = node1.getblock(node1.getbestblockhash())['tx'][0]
         locks0 = node0.gettxchainlocks([tx0, tx1])
         locks1 = node1.gettxchainlocks([tx0, tx1])
-        unknown_cl_helper = {'height': -1, 'chainlock': False}
-        assert_equal(locks0, [{'height': height, 'chainlock': True}, unknown_cl_helper])
-        assert_equal(locks1, [unknown_cl_helper, {'height': height1, 'chainlock': False}])
+        unknown_cl_helper = {'height': -1, 'chainlock': False, 'mempool': False}
+        assert_equal(locks0, [{'mempool': False, 'height': height, 'chainlock': True}, unknown_cl_helper])
+        assert_equal(locks1, [unknown_cl_helper, {'mempool': False, 'height': height1, 'chainlock': False}])
 
+        tx3 = node0.sendtoaddress(node0.getnewaddress(), 1)
+        locks0 = node0.gettxchainlocks([tx3])
+        locks1 = node0.gettxchainlocks([tx3])
```

Comment:
should it be `node1` here? looks strange to have locks0 == locks`

```suggestion
        self.sync_mempools()
        locks1 = node1.gettxchainlocks([tx3])
```
maybe?

Reply:
yes typo error. thanks!